src/style.css

```diff
@@ -0,0 +1,153 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+  font-style: normal;
+}
+
+html {
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 300;
+  font-size: 16;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.big-search-form {
+  width: 100%;
+  margin-top: 20px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 70px;
+  position: relative;
```

Comment:
Pay attention to CSS properties order
Check yourself everywhere
https://9elements.com/css-rule-order/

Reply:
learn next time!